client/components/meta-inputs/meta-datetime/Edit/index.vue

```diff
@@ -6,7 +6,7 @@
       :label="meta.label"
       :placeholder="meta.placeholder" />
     <time-picker
-      v-if="date && (meta.type === 'DATETIME')"
+      v-if="date && !meta.hideTime"
```

Comment:
Something that I like to do is split these conditions if one of them never (or rarely changes). In this case that would be the `meta.hideTime`. I would wrap this `time-picker` component with a template tag and toggle it with the `meta.hideTime` condition.